evap/static/ts/src/csrf-utils.ts

```diff
@@ -25,35 +25,38 @@ function isUrlCrossDomain(url: string): boolean {
 // setup ajax sending csrf token
 $.ajaxSetup({
     beforeSend: function (xhr: JQuery.jqXHR, settings: JQuery.AjaxSettings) {
-        const isMethodSafe = settings.method && isMethodCsrfSafe(settings.method);
-        if (!isMethodSafe && !this.crossDomain) {
+        const methodRequiresCsrfToken = settings.type && doesMethodRequireCsrfToken(settings.type);
```

Comment:
weirdly, settings.method was undefined in the js tests, only settings.type was set, which is supposed to be an alias to method. I did not dig into this further.

Reply:
@pixunil any idea why this would be the case?